data/person/01K5ST40KRR9V0N8N278NRWZ2S.toml

```diff
@@ -3,8 +3,9 @@ congress_website_primary_keys = [ 324,]
 congress_website_author_keys = [ "0012",]
 last_name = "Amante"
 first_name = "Edelmiro"
-middle_name = "A"
+middle_name = "Atega"
 aliases = [ "Edel",]
+name_suffix = "Sr."
```

Comment:
nit: we may need to change "Sr." to "Sr" to remove the periods to keep it consistent